engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -33,6 +33,7 @@
 import java.nio.file.Path;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import org.terasology.rendering.opengl.fbms.DynamicFBM;
 
 /**
  * The term "Post Processing" is in analogy to what occurs in the world of Photography:
```

Comment:
This beautiful comment that I wrote with so much love.... is no longer relevant. Sigh.

Really, the whole class it seems to me only takes care of the screenshot now. How about we rename it to ScreenGrabber?